plugins/ecs-bridge/main.go

```diff
@@ -47,8 +47,7 @@ func main() {
 
 	if printVersion {
 		if err := printVersionInfo(); err != nil {
-			os.Stderr.WriteString(
-				fmt.Sprintf("Error getting version string: %s", err.Error()))
+			fmt.Printf("Error getting version string: %s", err.Error())
```

Comment:
yes, write to the stderr won't be visible to the [err](https://github.com/aws/amazon-ecs-agent/blob/enis/agent/ecscni/plugin.go#L185). Write to the stdout will makes it captured by the err.